src/open_inwoner/conf/ci.py

```diff
@@ -13,7 +13,7 @@
     {
         "django": {
             "handlers": ["django"],
-            "level": "WARNING",
+            "level": "ERROR",
```

Comment:
I'm having doubts about the logging levels. 

Maybe we should move that to another issue en get the config changes and parallel test merged.

Reply:
I agree